managed/ui/src/components/config/Security/certificates/Certificates.js

```diff
@@ -176,12 +178,22 @@ class Certificates extends Component {
           <i className="fa fa-download"></i> Download Root CA Cert
         </MenuItem>
         <MenuItem
-          onClick={() => { 
-            this.deleteCertificate(payload?.uuid)
+          onClick={() => {
+            this.deleteCertificate(payload?.uuid);
           }}
           disabled={deleteDisabled}
+          title={deleteDisabled ? 'In use certificates cannot be deleted' : null}
+        >
+          <i className="fa fa-trash"></i> Delete Certificate
+        </MenuItem>
+        <MenuItem
+          onClick={() => {
+            // this.getAssociatedUniverseList(payload?.universeDetails);
+            this.setState({ associatedUniverses: [...payload?.universeDetails] })
+            this.props.showassociatedUniversesModal();
```

Comment:
same here - mixing local state and redux store usage

Reply:
Same as above